src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -67,11 +67,11 @@ private LogzioSender(String logzioToken, String logzioType, int drainTimeout, in
                         boolean addHostname, String additionalFields, int gcPersistedQueueFilesIntervalSeconds)
             throws IllegalArgumentException {
 
-        this.logzioToken = logzioToken;
+        this.logzioToken = getValueFromSystemEnvironmentIfNeeded(logzioToken);
```

Comment:
Can you please document this field support env variables and how in readme?
For both fields you've modified.